apps/admin-gui/src/assets/config/defaultConfig.json

```diff
@@ -22,6 +22,7 @@
     "oauth_response_type": "code",
     "oauth_offline_access_consent_prompt": true
   },
+  "proxy_logout": true,
```

Comment:
Should we also add new config property to this repo?
 https://github.com/CESNET/ansible-role-perun-docker-server

Reply:
Done: https://github.com/CESNET/ansible-role-perun-docker-server/pull/69